workflow/dumping-workflows.rst

```diff
@@ -9,6 +9,7 @@ them as SVG or PNG images. First, install any of these free and open source
 applications needed to generate the images:
 
 * `Graphviz`_, provides the ``dot`` command;
+* `Mermaid CLI`, provides the ``mmdc``command;
 * `PlantUML`_, provides the ``plantuml.jar`` file (which requires Java).
 
```

Comment:
We also tell people about new features using a `versionadded` block:
```suggestion

.. versionadded:: 5.3

    The ``mermaid`` dump format was introduced in Symfony 5.3.
```

Reply:
While I fixed the other issues, I left out this one as the feature PR is not yet merged and although there's still plenty of time until 5.3 I erred on the cautious side.